docker-compose.link.yaml

```diff
@@ -0,0 +1,4 @@
+networks:
+  default:
+    external:
+      name: fcp-fd
```

Comment:
Instead of this link file, you can specify the network in the docker compose file for each `service`.

Simplifies running locally greatly.

Reply:
Ahh, I think I added this in because I thought we were meant to have it, but can see this in the `docker-compose.override.yaml` file - I'm assuming this is the same `network`?:
```
networks:
  fcp-fd-comms:
    name: fcp-fd-comms
    driver: bridge
```
Please correct me if I'm wrong. Thank you.

Reply:
Set the network as `fcp-fd`.  

Also under each service in `docker-compose.override.yaml` there should be a `networks` property to bind the two together.

Example -> https://github.com/DEFRA/ffc-template-node/blob/main/docker-compose.override.yaml

(This is a bit misleading as the template uses the repo name, but it really should be a common network for all your services, so `fcp-fd` is enough for your needs)

Then that's the achieving the same as you have in the link file.